### DISCUSSION TITLE
SDV values for orthotropic bamboo damage not correctly reported

### DISCUSSION THREAD

**Discussion by JAWilliams123** [thelfer/MFrontGenericInterfaceSupport#141] [Q&A]
Hello Everyone,

I have developed the attached mfront to account for damage in bamboo based on Hill 1948 yield criteria. I was able to compile things and get it to run via Calculix, but for some strange reason, the damage state variables (dx, dy and dz) do not behave as expected. In the attached file, the damage has been hardwired to 0.1 but this is not what is shown the Calculix results. The single element CauliX input file has also been uploaded. Any assistance with this is greatly appreciated.
[BambooCalibrateDynamic.inp.txt](https://github.com/user-attachments/files/18281408/BambooCalibrateDynamic.inp.txt)
[HillYield01JW.mfront.txt](https://github.com/user-attachments/files/18281409/HillYield01JW.mfront.txt)

Many thanks and happy holidays,
JW

**Comment by thelfer** [comment]
@JAWilliams123 I close du to https://github.com/thelfer/tfel/discussions/678
